### DISCUSSION TITLE
graph: add timestamp to QFVote

### DISCUSSION THREAD

**Discussion by thelostone-mc** [gitcoinco/grants-stack#876] [Q&A]
Defer to  zen to add in more context here on usecase

Currently our while graph indexing does not record timestamp of the event fired. 
This is an improvement which would help FDD (check description from zen below)
This is a feature request to enable indexing timestamp of vote 

@nfrgosselin

**Comment by zen-sd** [comment]
We are relying heavily on the graph to easily pull data for analysis
and having the timestamps attached to the votes would help with certain types of analysis

This is also useful to make more visualizations on the dashboard

I had added an issue for this here https://github.com/gitcoinco/grants-round/issues/862

_**Projects applications** also would benefit from the timestamps_
